sklearnex/linear_model/logistic_regression.py

```diff
@@ -160,6 +162,16 @@ def predict_log_proba(self, X):
                 X,
             )
 
+        @wrap_output_data
+        def score(self, X, y, sample_weight=None):
+            if hasattr(y, "__sycl_usm_array_interface__"):
+                if hasattr(y, "__array_namespace__"):
+                    y = y.__array_namespace__().asnumpy(y)
+                else:
+                    y = y.asnumpy()
```

Comment:
I think it is better to generlize wrap_output_data or use usm_ndrarray_support instead of such conversions here